clang/lib/Basic/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(LLVM_LINK_COMPONENTS
   Support
   TargetParser
   FrontendOpenMP
+  FrontendDriver
```

Comment:
Ignore the earlier comment I wrote here I misread CodeGenOptions.cpp.

This is currently important into the clang Driver library which makes sense based on its name. Pulling it into Basic violates that name.

Reply:
I agree. I think I'm coming around to the opinion that this is a bit overkill just to avoid duplicating 16 lines of code.

I'm also thinking that flang and clang don't need to behave exactly the same in this regard. So we don't necessarily need to keep the two in sync. Thoughts? cc @tschuett

Reply:
I would prefer putting it into FrontendDriver for style and as a precedence to encourage others to put more into the frontend.

Reply:
I think the issue with llvmFrontendDriver specifically is that this code wouldn't be called from any of the drivers, it would be called from clangBasic and flangFrontend respectively. Could we create something like llvmFrontendSupport or similar?